docs/release-notes/release-notes-0.19.0.md

```diff
@@ -224,10 +225,29 @@ The underlying functionality between those two options remain the same.
 
 
 ## Breaking Changes
+
 ## Performance Improvements
 
 * Log rotation can now use ZSTD
 
+## Deprecations
+
+### ⚠️ **Warning:** The following RPCs will be removed in release version **0.21**:
+
+| Deprecated RPC Method | REST Equivalent | HTTP Method | Path | Replaced By |
+|----------------------|----------------|-------------|------------------------------|------------------|
+| [`lnrpc.SendToRoute`](https://lightning.engineering/api-docs/api/lnd/lightning/send-to-route/index.html) <br> [`routerrpc.SendToRoute`](https://lightning.engineering/api-docs/api/lnd/router/send-to-route/) | ❌ (No direct REST equivalent) | — | — | [`routerrpc.SendToRouteV2`](https://lightning.engineering/api-docs/api/lnd/router/send-to-route-v2/) |
+| [`lnrpc.SendPayment`](https://lightning.engineering/api-docs/api/lnd/lightning/send-payment/) <br> [`routerrpc.SendPayment`](https://lightning.engineering/api-docs/api/lnd/router/send-payment/) | ✅ | `POST` | `/v1/channels/transaction-stream` | [`routerrpc.SendPaymentV2`](https://lightning.engineering/api-docs/api/lnd/router/send-payment-v2/index.html) |
+| [`lnrpc.SendToRouteSync`](https://lightning.engineering/api-docs/api/lnd/lightning/send-to-route-sync/index.html) | ✅ | `POST` | `/v1/channels/transactions/route` | [`routerrpc.SendToRouteV2`](https://lightning.engineering/api-docs/api/lnd/router/send-to-route-v2/) |
+| [`lnrpc.SendPaymentSync`](https://lightning.engineering/api-docs/api/lnd/lightning/send-payment-sync/index.html) | ✅ | `POST` | `/v1/channels/transactions` | [`routerrpc.SendPaymentV2`](https://lightning.engineering/api-docs/api/lnd/router/send-payment-v2/index.html) |
```

Comment:
This table looks great, thanks!
One more candidate we should add here so we can remove it in 0.21 is [`TrackPayment`](https://lightning.engineering/api-docs/api/lnd/router/track-payment/) that has a `*V2` variant as well.
Then we're good to go!